modules/cluster_estimation/detection_in_world.py

```diff
@@ -0,0 +1,5 @@
+# DetectionInWorld class pending geolocation input
```

Comment:
Please rebase your branch on main or merge changes from main to your branch to get the data structures.